### DISCUSSION TITLE
[5.0] [Feature request] Grouping entries from the action log when sending by email

### DISCUSSION THREAD

**Discussion by Kostelano** [joomla/joomla-cms#47134] [2. Cold Features]
### Steps to reproduce the issue
1. Configure and check the efficiency of mail on the site.
2. Enable plugins related to user activity log.
3. In the administrator profile, enable sending entries from the action log by email.
4. Create a user and give him administrator rights.
5. Log in to the control panel as a new user and perform several actions (save / delete something).

You will receive a large number of notifications on the performed actions of the new user by mail. This is real spam.

I propose to revise the functionality in such a way as to add a parameter similar to the following content:

**Grouping before mail sending:** 1-2-3-4-5-10, etc.

It may seem to someone that the meaning of monitoring user actions is lost if, for example, you have chosen a grouping of 10 records, but this decision is solely for the administrator + the current state when you receive a lot of notifications for 1 record is terrible. 

----------

~Also, when translating language strings, I found that there was a string in the file that was in doubt. Firstly, it is the only file in which there is a separate line for the HTML-version of the email template.~

`COM_ACTIONLOGS_EMAIL_HTMLBODY="<h1>Latest User Actions</h1>\n<h2>This is the latest action performed by a user on your website.</h2>\n<table>\n<thead>\n<th>Action</th>\n<th>Date</th>\n<th>Extension</th>\n<th>Name</th>\n</thead>\n<tbody>\n{MESSAGES}<tr>\n<td>{MESSAGE}</td>\n<td>{DATE}</td>\n<td>{EXTENSION}</td>\n<td>{USERNAME}</td>\n</tr>{/MESSAGES}\n</tbody>\n</table>"`

~Why does it have multiple {MESSAGES} tags?~

**Comment by richard67** [comment]
> Why does it have multiple {MESSAGES} tags?

@Kostelano I see no multiple. I see one time an opening `{MESSAGES}` (plural), one time `{MESSAGE}` (singular, which will be replaced), and then a closing `{/MESSAGES}` (plural).

**Comment by Kostelano** [comment]
@richard67 Thank you. My carelessness. Sorry. I crossed out this question.
